firmware/application/freqman_db.cpp

```diff
@@ -74,12 +74,9 @@ options_t freqman_bandwidths[4] = {
     },
     {
         // SPEC -- TODO: these should be indexes.
-        {"12k5", 12500},
-        {"16k", 16000},
```

Comment:
This change affects 8 different apps.  Are you sure we want to remove these bandwidth options for all of them?

Reply:
Hi @NotherNgineer , 
Now,  I have clear understanding that regarding low bit rates, we have another constriction beside 
M4 % usage margin ,  SD card speed interface , proper usage of the anti-aliasing filters  .  
At the moment , we are struggling with Anti-alias filters , and therefore I want to simplify  the low bit rate options. 

Anyway,   I think those  Capture BW options are only used in Capture App. 
(but if I am not right , pls let me know about it .)

Reply:
The Level, Recon, and Spectrum Painter apps all support Spec mode and those bandwidth options.  I'm not sure how much it matters if we're taking away BW options that have existed for a long time.

I had additionally worried that BW index values may be stored by firmware <=1.7.4 in App Settings files and will no longer correspond to the same BW value if the App Settings are subsequently loaded by new firmware after this change.  But it looks like Spec mode BW values are NOT stored in App Settings, so that should not be a concern.

Reply:
I do think it probably matters in Spectrum Painter that we don't transmit with a wider bandwidth than necessary.